website/src/pages/news.tsx

```diff
@@ -0,0 +1,103 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import { ContentSection } from '../components/content/ContentSection'
+import Layout from '../components/Layout'
+export default class NewsPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            events: [],
+        }
+    }
+
+    public componentDidMount(): void {
+        if (document) {
+            document.getElementsByTagName('body')[0].setAttribute('style', 'background-image:none')
+        }
+    }
+
+    public render(): JSX.Element | null {
+        const desc =
+            'Sourcegraph in the news.'
+        return (
+            <Layout location={this.props.location}>
+                <div className="News bg-white text-dark">
+                    <Helmet>
+                        <title>Sourcegraph - News</title>
+                        <meta name="twitter:title" content="Sourcegraph news" />
+                        <meta property="og:title" content="Sourcegraph news" />
+                        <meta name="description" content="Sourcegraph news" />
+                        <meta name="twitter:description" content="Sourcegraph news" />
+                        <meta property="og:description" content="Sourcegraph news" />
+                    </Helmet>
+                    <section className="news news__head">
```

Comment:
Did you mean to include a stylesheet for news styles?

Reply:
Yes, eventually as the page evolves.  I would like help with compiling SASS.  what should I use?

Reply:
The SASS files are compiled for you automatically when running the dev server and building and to add a new stylesheet for a page, add a file to `website/src/css/pages` and then include a link to that file in `website/src/css/styles.scss`.